rbac/management/cache.py

```diff
@@ -49,15 +50,31 @@ def get_policy(self, uuid, application):
             logger.exception("Error querying policy for uuid %s", uuid)
         return None
 
-    def delete_policy(self, uuid):
-        """Purge the given user's policy from the cache."""
+    @contextlib.contextmanager
+    def delete_handler(self, err_msg):
+        """Handle policy delete events."""
         if not settings.ACCESS_CACHE_ENABLED:
             return
         try:
+            yield
+        except exceptions.RedisError:
+            logger.exception(err_msg)
+
+    def delete_policy(self, uuid):
+        """Purge the given user's policy from the cache."""
+        err_msg = "Error deleting policy for uuid %s", uuid
```

Comment:
This would be a tuple instead of string, but seems it is working?

Reply:
@astrozzc it does work, yes, but I added a change in 6ea9c03 which should clean it up. 😄